---
title: Getting Started in the Jin Lab and learning about DNA Inversions
layout: single
author: hwang-ayden
excerpt: Ayden's HYRS Experience and Genome Inversions!

---

Like how we can flip the word “melon” into “lemon,” bacterial genomes can undergo the same process with their GCA and Ts! In DNA, these regions are known as invertons, and recombinase enzymes bind to the DNA and flip the inverton. However, these enzymes do not bind and flip the inverton itself; conversely, recombinases bind to regions known as invertons that flank the inverton. These inverted repeats are flipped in a way that is similar to flipping the word “radar” to “radar.” In other words, inverted repeats remain the same while invertons flip and change, and ultimately, this flipping of the inverton regulates gene expression, turning a melon into a lemon. 

**My Lab Experience**

I took genome data from a synthetic microbial community, hCom2, grown in an environment in and without the presence of streptomycin. With this data, I ran the sequenced reads through a bioinformatics workflow tool, PhaseFinderDC, which detected invertons and their respective counts. With the resulting data, I used statistical tests like Scipy Fisher’s exact test and MatPlotLib and Pandas to generate plots representing that streptomycin leads to inversions. Furthermore, by using IGV (Integrative Genomics Viewer), I noticed that besides these inventions were regulatory elements like recombinase,




